.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -1,15 +1,11 @@
----
-name: "Documentation \U0001F516"
-about: Issues and feedback related to the docs.
-title: "[DOC] - <title>"
-labels: ["type: documentation"]
+______________________________________________________________________
 
----
+name: "Documentation \\U0001F516" about: Issues and feedback related to the docs. title: "\[DOC\] - <title>" labels: \["type: documentation"\]
+
+______________________________________________________________________
```

Comment:
This might need to be refactored again, not sure if Github will accept as a valid template

Reply:
Good spot, I will restrict to docs only for now, and revert to earlier files elsewhere.